apps_download/models/product_product.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2017-Today: Odoo Community Association (OCA)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+import os
+import tempfile
+import shutil
+import logging
+import base64
+import subprocess
+import time
+from odoo.exceptions import ValidationError
+from odoo import models, fields, api, _
+_logger = logging.getLogger(__name__)
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    dependent_product_ids = fields.Many2many(
+        'product.product', 'product_product_dependent_rel',
+        'src_id', 'dest_id', string='Dependent Products'
+    )
+    module_path = fields.Char('Module Path')
```

Comment:
If the string is the same that the name of the field is not necessary define the string because is used by default.

3rd point from https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#fields